microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -149,15 +153,12 @@ if (sendMessageId) {
 
 ---
 
-
 <!-- ====================================================================== -->
 ## Step 4: Make your `stars.jpeg` available from any browser tab
 
-When you pass the `images/stars.jpeg`, you must use the `chrome.runtime.getURL` API (or `chrome.extension.getURL` if using Manifest V2) instead of just passing in the relative URL without the extra prefix like in the previous section.  That extra prefix is returned by `getUrl` with the image attached, and looks something like the following:
+To make `images/stars.jpeg` available from any browser tab, you must use the `chrome.runtime.getURL` API.  
 
-```http
-extension://inigobacliaghocjiapeaaoemkjifjhp/images/stars.jpeg
-```
+Note: If you are using Manifest V2, then instead use `chrome.extension.getURL`.  That extra prefix is returned by `getUrl` with the image attached, and looks something like the following: ```httpextension://inigobacliaghocjiapeaaoemkjifjhp/images/stars.jpeg```
```

Comment:
```suggestion
Note: If you are using Manifest V2, then instead use `chrome.extension.getURL`.  That extra prefix is returned by `getURL` with the image attached, and looks something like the following: ```httpextension://inigobacliaghocjiapeaaoemkjifjhp/images/stars.jpeg```
```
check casing.
I'd try to minimize use of "Note: "